### ISSUE TITLE
Remove GCM-specific settings when they will no longer be needed

### ISSUE THREAD

**Issue by marco-c** [mozilla/wp-web-push#216]
https://bugs.chromium.org/p/chromium/issues/detail?id=583753

**Comment by mintplugins** [comment]
@marco-c Is this project still being maintained? If not, is there a place you'd recommend looking for a replacement?

**Comment by marco-c** [comment]
No, unfortunately it's no longer maintained. I don't know of any replacement.
